src/common/models/immutable.go

```diff
@@ -0,0 +1,15 @@
+package models
+
+// ImmutableRule - rule which filter image tags should be immutable.
+type ImmutableRule struct {
+	ID         int64  `orm:"pk;auto;column(id)" json:"id,omitempty"`
+	ProjectID  int64  `orm:"column(project_id)" json:"project_id,omitempty"`
+	RepoFilter string `orm:"column(repo_filter)" json:"repo_filter,omitempty"`
+	TagFilter  string `orm:"column(tag_filter)" json:"tag_filter,omitempty"`
```

Comment:
How about defining a filter list that can contain kinds of different filters？In this way, the supported filters can be extended easily

Reply:
Remove the repoFilter, all information store ini tagfilter

Reply:
Remove repo filter, all filter stored in tagfilter